### PR TITLE
Metadata: isaaclevin.presencelight version 3.0.19

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/21823)

### CODE REVIEW

File: manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 3.0.19
+MinimumOSVersion: 10.0.0.0
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.installer.yaml:5]:
```suggestion
PackageVersion: 3.0.19.0
MinimumOSVersion: 10.0.17134.0
```

File: manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
+  InstallerUrl: https://github.com/isaacrlevin/PresenceLight/releases/download/v3.0.19/PresenceLight.Package_3.0.19.0_x86.appxbundle
+  InstallerSha256: 4479FB300FEA553E421BDBB318984E3AF3357E8A7642ECD0F381DFE8F978878C
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.installer.yaml:16]:
```suggestion
  InstallerSha256: 4479FB300FEA553E421BDBB318984E3AF3357E8A7642ECD0F381DFE8F978878C
  SignatureSha256: 43EA3D0B2AEB1EA8D85B86675F0205A16C2839E999FCA40AE5035DE67317652E
  PackageFamilyName: 37828IsaacLevin.197278F15330A_jvewcxq8vj8qt
```

File: manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: isaaclevin.presencelight
+PackageVersion: 3.0.19
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 3.0.19.0
```

File: manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.yaml [hunk 4]
```diff
@@ -1,19 +1,8 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
 PackageIdentifier: isaaclevin.presencelight
 PackageVersion: 3.0.19
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/3.0.19/isaaclevin.presencelight.yaml:4]:
```suggestion
PackageVersion: 3.0.19.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/i/isaaclevin/presencelight/3.0.19.0/isaaclevin.presencelight.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+MinimumOSVersion: 10.0.17134.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: appx
+  Scope: machine
```
Comment (OfficialEsco) [inline manifests/i/isaaclevin/presencelight/3.0.19.0/isaaclevin.presencelight.installer.yaml:14]:
```suggestion
Installers:
- InstallerLocale: en-US
  Architecture: x86
  InstallerType: appx
  Scope: user
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

**Review by OfficialEsco — DISMISSED** [review]

**Comment by ItzLevvie** [comment]
cc @denelon @palenshus: "Azure-Pipeline-Passed" label needs to be added to this PR.

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**